circle.yml

```diff
@@ -9,7 +9,7 @@ dependencies:
     - if [[ ! -d mongodb-linux-x86_64-3.2.4 ]]; then wget http://downloads.mongodb.org/linux/mongodb-linux-x86_64-3.2.4.tgz && tar xvzf mongodb-linux-x86_64-3.2.4.tgz; fi
     - sudo /etc/init.d/mongod stop
     - sudo cp mongodb-linux-x86_64-3.2.4/bin/* /usr/bin
-    - sudo /etc/init.d/mongod start
+    - sudo /etc/init.d/mongod start --port 27018
```

Comment:
Maybe we can speedup the circleci build too with this version: https://docs.mongodb.com/manual/core/inmemory/ (ie. by adding the `--storageEngine` parameter)